.github/workflows/canary-integration-test.yml

```diff
@@ -884,4 +884,177 @@ jobs:
+        kubectl create -f cluster-test.yaml -f rbdmirror.yaml -f filesystem-mirror.yaml -f toolbox.yaml
+
+    - name: wait for ceph cluster 1 to be ready
+      run: |
+        mkdir test
+        tests/scripts/validate_cluster.sh osd 1
+        kubectl -n rook-ceph get pods
+
+    - name: create replicated mirrored pool on cluster 1
+      run: |
+        cd cluster/examples/kubernetes/ceph/
+        yq w -i pool-test.yaml spec.mirroring.enabled true
+        yq w -i pool-test.yaml spec.mirroring.mode image
+        kubectl create -f pool-test.yaml
+        timeout 60 sh -c 'until [ "$(kubectl -n rook-ceph get cephblockpool replicapool -o jsonpath='{.status.phase}'|grep -c "Ready")" -eq 1 ]; do echo "waiting for pool to created" && sleep 1; done'
+
+    - name: create replicated mirrored pool on cluster 2
+      run: |
+        cd cluster/examples/kubernetes/ceph/
+        yq w -i pool-test.yaml metadata.namespace rook-ceph-external
+        kubectl create -f pool-test.yaml
+        timeout 60 sh -c 'until [ "$(kubectl -n rook-ceph-external get cephblockpool replicapool -o jsonpath='{.status.phase}'|grep -c "Ready")" -eq 1 ]; do echo "waiting for pool to created" && sleep 1; done'
+
+    - name: create image in the pool
+      run: |
+        kubectl exec -n rook-ceph deploy/rook-ceph-tools -ti -- rbd -p replicapool create test -s 1G
+        kubectl exec -n rook-ceph deploy/rook-ceph-tools -t -- rbd mirror image enable replicapool/test snapshot
+        kubectl exec -n rook-ceph deploy/rook-ceph-tools -t -- rbd -p replicapool info test
+
+    - name: copy peer secret into the other cluster
+      run: |
+        kubectl -n rook-ceph get secret pool-peer-token-replicapool -o yaml |\
+        sed 's/namespace: rook-ceph/namespace: rook-ceph-external/g; s/name: pool-peer-token-replicapool/name: pool-peer-token-replicapool-config/g' |\
+        kubectl create --namespace=rook-ceph-external -f -
+
+    - name: add peer secret to the other cluster
+      run: |
+        kubectl -n rook-ceph-external patch cephrbdmirror my-rbd-mirror --type merge -p '{"spec":{"peers": {"secretNames": ["pool-peer-token-replicapool-config"]}}}'
+
+    - name: verify image has been mirrored
+      run: |
+        # let's wait a bit for the image to be present
+        timeout 30 sh -c 'until [ "$(kubectl exec -n rook-ceph-external deploy/rook-ceph-tools -t -- rbd -p replicapool ls|grep -c test)" -eq 1 ]; do echo "waiting for image to be mirrored" && sleep 1; done'
+
+    - name: display cephblockpool and image status
+      run: |
+        timeout 80 sh -c 'until [ "$(kubectl -n rook-ceph-external get cephblockpool replicapool -o jsonpath='{.status.mirroringStatus.summary.daemon_health}'|grep -c OK)" -eq 1 ]; do echo "waiting for mirroring status to be updated" && sleep 1; done'
+        kubectl -n rook-ceph-external get cephblockpool -o yaml
+        kubectl exec -n rook-ceph deploy/rook-ceph-tools -t -- rbd -p replicapool info test
+
```

Comment:
We'll be able to use this same action for cephfs mirroring, right? At least it seems like a small add-on

Reply:
Yes, it's the goal.